src/WebJobs.Script/Workers/Rpc/Configuration/RpcWorkerConfigFactory.cs

```diff
@@ -213,61 +231,6 @@ internal void AddProvider(string workerDir)
             }
         }
 
-        private static JsonElement GetWorkerConfigJsonElement(string workerConfigPath)
```

Comment:
These methods are also moved to class `WorkerConfigurationHelper` as we will need to call these methods from class `WorkerConfigurationResolver` as part of compatibility check.